packages/@aws-cdk/aws-kinesis/README.md

```diff
@@ -17,34 +17,86 @@
 ---
 <!--END STABILITY BANNER-->
 
-Define an unencrypted Kinesis stream.
+[Amazon Kinesis](https://docs.aws.amazon.com/streams/latest/dev/introduction.html) provides collection and processing of large [streams](https://aws.amazon.com/streaming-data/) of data records in real time. Kinesis data streams can be used for rapid and continuous data intake and aggregation.
+
+## Table Of Contents
+- [Streams](#streams)
+  - [Encryption](#encryption)
+  - [Import](#import)
+
+
+### Streams
```

Comment:
Use h2 instead of h3. Or, switch 'table to contents' to h3 as well.

I would suggest the former.

Reply:
done